src/controllers/user.js

```diff
@@ -71,13 +71,13 @@ const getUserProducts = async (req, res) => {
 
 const getSingleUser = async (req, res) => {
   try {
-    const foundUser = await UserModel.findById(req.params.id);
-    if (foundUser) {
-      if (req.user?.userId === foundUser.id) {
+    const User = await UserModel.findById(req.params.id);
+    if (User) {
+      if (req.user?.userId === User.id) {
         return res.status(200).json({ message: 'Redirecting to profile...' });
       }
 
-      const shownInfo = { ...foundUser._doc, passwordHash: null };
+      const shownInfo = { ...User._doc, passwordHash: null };
```

Comment:
can you change this _doc part to User.toObject()? it is a better approach i assume

Reply:
@okurezgi that was my merge 😆 it has noting to do with @awiednoor. That's how Halit requested it, for it to not show the hash. So Noor doesn't really need to change that. In fact, that'd affect tests.